src/Makefile.in

```diff
@@ -129,6 +129,7 @@ endif
 
 ifeq ($(STATIC),1)
 	LDFLAGS+=$(addsuffix .a,$(addprefix -l:lib,$(LIBS)))
+	LDFLAGS+=-static
```

Comment:
I noticed this and added it in https://github.com/catalinii/minisatip/pull/1271 too, I think we'll merge your PR before mine so let's keep this here